pom.xml

```diff
@@ -30,19 +30,8 @@
 
   <developers>
     <developer>
-      <id>mgalesloot</id>
-      <name>Matthijs Galesloot</name>
-      <email>matthijs@codehaus.org</email>
-    </developer>
-    <developer>
-      <id>henryju</id>
-      <name>Julien Henry</name>
-      <email>julien.henry@sonarsource.com</email>
-    </developer>
-    <developer>
-      <id>lindamartin</id>
-      <name>Linda Martin</name>
-      <email>linda.martin@sonarsource.com</email>
+      <id>pynicolas</id>
```

Comment:
are developers mandatory? I feel it's a bit too much to add PY. I would just drop this from the pom.